src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -745,6 +762,16 @@ internal void ThrowIfFunctionDoesNotExist(string name, FunctionType functionType
             }
         }
 
+        private static bool IsDurableHttpTask(string functionName)
+        {
+            if (functionName == HttpOptions.HttpTaskActivityReservedName)

```

Comment:
You can make this function one line by just doing `return string.Equals(functionName, HttpOptions.HttpTaskActivityReservedName)`.